### ISSUE TITLE
Upgrade to Hyrax v3.0.0.pre.rc1 / rails v5.2

### ISSUE THREAD

**Issue by fritzfreiheit** [samvera-labs/samvera-persona#16]
The following exception occurred while trying to run `rake -T`

```
> bundle exec rake -T
NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
Gem::Specification#rubyforge_project= called from .rbenv/versions/2.4.10/lib/ruby/gems/2.4.0/specifications/em-websocket-0.5.1.gemspec:15.
rake aborted!
NoMethodError: undefined method `merge!' for nil:NilClass
.bundle/ruby/2.4.0/gems/railties-5.2.4.3/lib/rails/configuration.rb:132:in `method_missing'
.bundle/ruby/2.4.0/gems/samvera-persona-0.1.7/lib/samvera-persona/engine.rb:23:in `block in <class:Engine>'
.bundle/ruby/2.4.0/gems/railties-5.2.4.3/lib/rails/engine/configuration.rb:33:in `generators'
.bundle/ruby/2.4.0/gems/samvera-persona-0.1.7/lib/samvera-persona/engine.rb:22:in `<class:Engine>'
.bundle/ruby/2.4.0/gems/samvera-persona-0.1.7/lib/samvera-persona/engine.rb:7:in `<module:Persona>'
.bundle/ruby/2.4.0/gems/samvera-persona-0.1.7/lib/samvera-persona/engine.rb:6:in `<module:Samvera>'
.bundle/ruby/2.4.0/gems/samvera-persona-0.1.7/lib/samvera-persona/engine.rb:5:in `<top (required)>'
.bundle/ruby/2.4.0/gems/activesupport-5.2.4.3/lib/active_support/dependencies.rb:291:in `require'
.bundle/ruby/2.4.0/gems/activesupport-5.2.4.3/lib/active_support/dependencies.rb:291:in `block in require'
.bundle/ruby/2.4.0/gems/activesupport-5.2.4.3/lib/active_support/dependencies.rb:257:in `load_dependency'
.bundle/ruby/2.4.0/gems/activesupport-5.2.4.3/lib/active_support/dependencies.rb:291:in `require'
.bundle/ruby/2.4.0/gems/samvera-persona-0.1.7/lib/samvera-persona.rb:6:in `<top (required)>'
config/application.rb:11:in `<top (required)>'
Rakefile:4:in `require_relative'
Rakefile:4:in `<top (required)>'
.bundle/ruby/2.4.0/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
.rbenv/versions/2.4.10/bin/bundle:23:in `load'
.rbenv/versions/2.4.10/bin/bundle:23:in `<main>'
(See full trace by running task with --trace)
```

**Comment by fritzfreiheit** [comment]
After a suggestion from @orangewolf to try the branch `remove-generator-config`, the above error was fixed. That is, by removing lines 22-27 of `lib/samvera-persona/engine.rb` , the nil exception no longer occurs when running our upgrade to Rails 5.2 / Hyrax 3.0.0.pre.rc1 with `samvera-persona` gem.

**Comment by orangewolf** [comment]
This was released as v0.2.0
